components/navbar/ProfileDropdown.tsx

```diff
@@ -51,22 +51,24 @@ export const ProfileDropdown: FC = () => {
           e.preventDefault()
         }}
       >
-        <CopyText
-          text={shortEnsName ? shortEnsName : shortAddress}
-          css={{ width: '100%' }}
-        >
-          <Flex justify="between" css={{ width: '100%' }}>
+        <Link href={`/profile/${address}`} style={{ flex: 1 }}>
+          <Flex justify="between" align="center" css={{ width: '100%' }}>
             <Text style="subtitle1" color="subtle">
               {shortEnsName ? shortEnsName : shortAddress}
             </Text>
-            <Box css={{ color: '$gray10' }}>
-              <FontAwesomeIcon icon={faCopy} width={16} height={16} />
-            </Box>
+            <CopyText
```

Comment:
@pedromcunha I think we want to get rid of the CopyText and copy icon if this is now a link

Reply:
Ah I see, looks good then!